metrics/metrics.go

```diff
@@ -153,6 +153,11 @@ func RegisterMetrics() {
 	prometheus.MustRegister(SmallTxnWriteDuration)
 	prometheus.MustRegister(TxnWriteThroughput)
 	prometheus.MustRegister(LoadSysVarCacheCounter)
+	prometheus.MustRegister(RegisterSQLFail)
```

Comment:
How about adding a `TopSQL` prefix to all those new variables?

Reply:
done.